src/main/java/org/spongepowered/api/item/Enchantment.java

```diff
@@ -112,4 +112,15 @@
      * @return Whether these enchantments are compatible
      */
     boolean isCompatibleWith(Enchantment ench);
+
+    /**
+     * Gets whether or not this enchantment is considered a "treasure"
+     * enchantment.
+     *
+     * <p>These do not occur naturally in enchantment tables.</p>
+     *
+     * @return If this is a treasure enchantment
+     */
+    boolean isTreasureEnchantment();
```

Comment:
I was trying to think of a better name since enchantment seemed redundant, but "isTreasure" doesn't really convey what it is that well.